### ISSUE TITLE
Small Error Regarding WaveDirMod

### ISSUE THREAD

**Issue by jjonkman** [OpenFAST/openfast#246]
HydroDyn input parameter WaveDirMod is supposed to be used only when WaveMod=2, 3, or 4.  However, the logic in routine HydroDyn_Input.f90/HydroDynInput_ProcessInitData() forces WaveMod to equal 0 if WaveDirMod == 1 and WaveMod <> 2, 3, or 4.  Instead of setting WaveMod = 0, WaveMultiDir should be set to .FALSE. in this case.

This issue was reported in the following topic on the forum: https://wind.nrel.gov/forum/wind/viewtopic.php?f=4&t=2213.

**Comment by ebranlard** [comment]
The logic in the code is as follows:
```fortran
WaveMultiDir = .FALSE.         ! Set flag to false to start
IF (WaveMod >= 2 .AND. WaveMod <= 4 .AND. WaveDirMod == 1 ) THEN
      WaveMultiDir = .TRUE.
ELSEIF ( (WaveMod < 2 .OR. WaveMod >4) .AND. WaveDirMod == 1 ) THEN
      CALL WARNING('WaveDirMod unused unless WaveMod == 2, 3, or 4.  Ignoring WaveDirMod.',)
      WaveMod   = 0
ENDIF
```

The last assignement should be:
```
 WaveDirMod   = 0
```

I'll submit a pull request

**Comment by jjonkman** [comment]
Actually, I think all you need to do is eliminate the last assignment (WaveMod = 0) altogether.

**Comment by rafmudaf** [comment]
Fixed by #247
